### PR TITLE
Provide a README and express sever to serve content

### PR DESCRIPTION
**Pull request by johnny-rice**
Checkout this branch and please install the latest libraries using `npm i` and check that the new /README.md file has the right instructions.

This branch moves most code to the app folder and adds a server.js file to create a server that delivers the app as a local web server.

Adds these scripts

`npm run dev` which will run  `node --watch ./server.js`,
  
`npm start` which is meant for deploying to production and runs `node ./server.js`

`npm run format:prettier` which updates the code to match the prettier rules. Standard code format is the way to easy collaboration.

When you run `npm run dev` from the terminal, the browser should server the content like this:

http://localhost:8100/

Screenshot 2023-12-07 at 5 14 13 PM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+# Project Name
```
Comment (johnny-rice) [inline README.md:1]: Please update this README.md file with any updates you see that will be useful.

File: app/calculateFunc.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+function calculateFunc(n1, operator, n2) {
```
Comment (johnny-rice) [inline app/calculateFunc.js:1]: The existing code file was just moved to the app folder.

File: app/calculateFunc.test.js [hunk 3]
```diff
@@ -0,0 +1,17 @@
+const calculateFunc = require("./calculateFunc");
```
Comment (johnny-rice) [inline app/calculateFunc.test.js:1]: The existing code file was just moved to the app folder.

File: app/index.html [hunk 4]
```diff
@@ -0,0 +1,36 @@
+<!doctype html>
```
Comment (johnny-rice) [inline app/index.html:1]: The existing code file was just moved to the app folder.

File: app/index.js [hunk 5]
```diff
@@ -0,0 +1,104 @@
+const calculator = document.querySelector(".calculator");
```
Comment (johnny-rice) [inline app/index.js:1]: Still! This is also existing code file was just moved to the app folder.

File: app/style.css [hunk 6]
```diff
@@ -1,4 +1,3 @@
-
```
Comment (johnny-rice) [inline app/style.css:1]: Also.. moved..

File: package.json [hunk 7]
```diff
@@ -4,12 +4,20 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "jest"
```
Comment (johnny-rice) [inline package.json:7]: I added these by typing this into this part of the package.json file.

This corresponds to running this command to install the packages:

`npm i express get-port`

and 

`npm i -D prettier`

**Comment by johnny-rice** [comment]
@ManhTrinhNguyen, if you will, please update any files you would like, such as the README. If all looks good after, please approve and merge. That action will make this code the latest in the main branch for this repo.

Then I suggest making a new branch and finding some changes to make. After the changes are made on your new branch, try out the PR process by creating a PR and adding me as a reviewer.

File: app/index.js [hunk 8]
```diff
@@ -0,0 +1,104 @@
...
+      calculator.dataset.firstValue = "";
+      calculator.dataset.modValue = "";
+      calculator.dataset.operator = "";
+      calculator.dataset.previousKeyType = "";
+
+      display.textContent = 0;
+      calculator.dataset.previousKeyType = "clear";
+    }
+
+    if (action === "calculate") {
```
Comment (johnny-rice) [inline app/index.js:86]: This existing code has some deep nesting. We can discuss the challenges this creates and the solutions available as part of a new effort later this week.

Comment (ManhTrinhNguyen) [reply app/index.js:86]: Ok i got it

**Pull request merged**